### PR TITLE
Mobile Responsive

### PR DESCRIPTION
**Pull request by parkerdavis1**
Changes to allow users to access most functionality on mobile including a simple fix for Nav Bar issue #2

### CODE REVIEW

**Comment by benhsm** [comment]
It may take me some time to review this; I'll have to ask you about some of these changes later on, either here or in the slack.

**Comment by parkerdavis1** [comment]
Sure! I probably should have broken it down into smaller chunks.

File: app/components/ui/dialog.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const DialogContent = React.forwardRef<
 		<DialogPrimitive.Content
 			ref={ref}
 			className={cn(
-				'fixed left-[50%] top-[50%] z-50 grid w-full max-w-lg translate-x-[-50%] translate-y-[-50%] gap-4 border bg-background p-6 shadow-lg duration-200 data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 data-[state=closed]:slide-out-to-left-1/2 data-[state=closed]:slide-out-to-top-[48%] data-[state=open]:slide-in-from-left-1/2 data-[state=open]:slide-in-from-top-[48%] sm:rounded-lg md:w-full',
+				'fixed left-[50%] top-[50%] z-50 grid w-full max-w-lg max-h-full overflow-scroll translate-x-[-50%] translate-y-[-50%] gap-4 border bg-background p-6 shadow-lg sm:rounded-lg duration-200 data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 data-[state=closed]:slide-out-to-left-1/2 data-[state=closed]:slide-out-to-top-[48%] data-[state=open]:slide-in-from-left-1/2 data-[state=open]:slide-in-from-top-[48%]',
```
Comment (parkerdavis1) [inline app/components/ui/dialog.tsx:43]: rearrangement. 
removed redundant md:w-full. 
max-h-full and overflow-scroll allows dialogs to be scrollable on smaller screens.

File: app/components/ui/dialog.tsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ const DialogHeader = ({
 }: React.HTMLAttributes<HTMLDivElement>) => (
 	<div
 		className={cn(
-			'flex flex-col space-y-1.5 text-center sm:text-left',
+			'flex flex-col space-y-1.5 text-left',
```
Comment (parkerdavis1) [inline app/components/ui/dialog.tsx:60]: simplification, aesthetic choice

File: app/components/ui/tooltip.tsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ const TooltipContent = React.forwardRef<
 	<TooltipPrimitive.Content
 		ref={ref}
 		sideOffset={sideOffset}
+		align={"end"}
```
Comment (parkerdavis1) [inline app/components/ui/tooltip.tsx:19]: improves tooltip visibility

File: app/root.tsx
Comment (parkerdavis1) [inline app/root.tsx:1]: some restructuring and tweaks to make the nav bar mobile responsive

File: app/routes/admin+/_users+/users.edit.$userId.tsx
Comment (parkerdavis1) [inline app/routes/admin+/_users+/users.edit.$userId.tsx:1]: input fields take up full width on <400px screens

**Comment by benhsm** [comment]
> Sure! I probably should have broken it down into smaller chunks.

Actually, would it be a lot of trouble for you to break out the navbar related changes into a different PR? I put that together pretty fast and loose on the first pass, it could use a deep refactor. I'd like for the mobile view of the navbar to have an icon link to the home page, or something, so the sprites may need to be regenerated.

File: app/routes/calendar+/$eventId.edit.tsx
Comment (parkerdavis1) [inline app/routes/calendar+/$eventId.edit.tsx:1]: input fields take up full width on <400px screens

**Comment by benhsm** [comment]
Looks good to me, nice work, and thanks for accommodating me re: the splitting and revisions.

**Pull request merged**